### ISSUE TITLE
Publish & send button has incorrect background colour in dark mode on hover

### ISSUE THREAD

**Issue by stromfeldt** [TryGhost/Ghost#13700]
### Issue Summary

The background colour is incorrect when hovering over the "Publish & send" button while in dark mode.

### To Reproduce

1. Place the admin in dark mode
2. Click on "Publish" to open the "Ready to publish and send this post?" box
3. Hover over the white on green "Publish & send" button
4. The button turns to white on very light grey

![Screen Shot 2021-10-29 at 2 13 32 pm](https://user-images.githubusercontent.com/27488257/139372110-1451d839-58a5-429c-9a6c-182d99fe6225.png)

### Technical details:

* Ghost Version: 4.20.3

**Comment by ibandim123** [comment]
Hi, let's me see... the color button his must by like that??
![image](https://user-images.githubusercontent.com/59544620/144073645-534569fb-eaf8-49a9-9cb7-b0ae0b7198b3.png)

**Comment by jyoung0519** [comment]
We are a group of 3 students looking to contribute to an open-source project for a university assignment. We would like to give this one a shot!

**Comment by kumarrus** [comment]
Hi, I would like to fix this issue. Can I take this bug?

I've looked into this issue and its because of conflicting CSS styling rules on hover for the two classes associated with the button `gh-btn-black` and `gh-publishmenu-button`:

```
.gh-btn-black:not(.gh-btn-green):not(.gh-btn-blue):not(.gh-btn-red):hover,
.gh-btn-primary:not(.gh-btn-green):not(.gh-btn-blue):not(.gh-btn-red):hover {
    background: var(--black) !important;
}

.gh-publishmenu-button:hover {
    color: #fff;
    background: var(--green) !important; --> This is ignored
}
```

The text color can be changed from `#fff` to something darker like `#394047` or `var(--white)`. But that will be inconsistent behavior with similar green buttons like LaunchWizard button. Ideally, the button should have `gh-btn-green` class in dark mode. **Should we make it green in light mode as well?**

![image](https://user-images.githubusercontent.com/2130096/148157467-4367dded-266d-4a30-a68a-094847e7e748.png)
![image](https://user-images.githubusercontent.com/2130096/148157516-95cb14cb-2dd7-41ea-a0fc-34157f360340.png)

**Comment by kumarrus** [comment]
@ErisDS , Any updates on this?

**Comment by ErisDS** [comment]
There was a PR for this merged in [v4.35.0](https://github.com/TryGhost/Admin/releases/tag/v4.35.0), but Ghost v5 has a whole new publishing flow.
